examples/go.mod

```diff
@@ -2,11 +2,4 @@ module github.com/pulumi/pulumi-civo/examples/v2
 
 go 1.16
 
-require (
-	github.com/onsi/ginkgo v1.12.0 // indirect
-	github.com/onsi/gomega v1.9.0 // indirect
-	github.com/pulumi/pulumi/pkg/v3 v3.0.0
-	github.com/pulumi/pulumi/sdk/v3 v3.0.0 // indirect
-	gopkg.in/airbrake/gobrake.v2 v2.0.9 // indirect
-	gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2 // indirect
-)
+require github.com/pulumi/pulumi/pkg/v3 v3.17.0
```

Comment:
do you also need `github.com/pulumi/pulumi/sdk/v3`? I notice that got removed here.

Reply:
No.  This only affects the examples, and although I'm not sure why b/c I don't understand well enough yet, at 3.17.0, only the pkg is needed, not the sdk.